File: modul_praktikum_python/contoh_code/pascal/pola_bintang.pas
uses crt; 
Var 
  N : integer; (* u *) 
  I,j: integer;(* counter *) 
begin 
 write(‘Masukan jumlah awal bintang : ‘);readln(N); 
 for i:=N downto 0 do 
  begin  
   for j:=1 to i do  
     write(‘*’); 
  writeln; 
  end;     
end.

File: modul_praktikum_python/contoh_code/python/pola_bintang.py
"""Segitiga bintang dengan baris mengecil."""


def main() -> None:
    n = int(input("Masukan jumlah awal bintang : "))
    for i in range(n, -1, -1):
        for j in range(1, i + 1):
            print("*", end="")
        print()


if __name__ == "__main__":
    main()
